### PR TITLE
[HW-3] classes

### PR DESCRIPTION
**Pull request by BukhtikYu**

### CODE REVIEW

**Comment by vladhilko** [comment]
Please, remove `hw1` and `hw2` from this PR

File: HW03/Yuliya_Bukhtik/student.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# this module is a subject's interface
+
+module Observable
```
Comment (vladhilko) [inline HW03/Yuliya_Bukhtik/student.rb:5]: Could you move this module to a separate file?

File: HW03/Yuliya_Bukhtik/homework.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# This is class Homework
+
+# frozen_string_literal: true
+
+require 'BaseObserver'
+
+class Homework < BaseObserver
```
Comment (B0RlS) [inline HW03/Yuliya_Bukhtik/homework.rb:7]: I think this is not true inheritance

File: HW03/Yuliya_Bukhtik/mentor.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# this class Mentor
+
+class Mentor
```
Comment (B0RlS) [inline HW03/Yuliya_Bukhtik/mentor.rb:5]: You must add attr_accessor to the variables

File: HW03/Yuliya_Bukhtik/notification.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'BaseObserver'
+
+class Notification < BaseObserver
```
Comment (B0RlS) [inline HW03/Yuliya_Bukhtik/notification.rb:5]: same problem

File: HW03/Yuliya_Bukhtik/student.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+require_relative 'observable'
+
+class Student
+  include Observable
+
+  attr_accessor :name, :surname, :homework
+
+  def initialize(name:, surname:)
+    super()
```
Comment (B0RlS) [inline HW03/Yuliya_Bukhtik/student.rb:11]: What is this for?

Comment (BukhtikYu) [reply HW03/Yuliya_Bukhtik/student.rb]: I deleted it, I added it by mistake.

**Pull request merged**